mariner/process.py

```diff
@@ -1,33 +1,52 @@
+import subprocess
 import tempfile
 import logging
-from mariner.util import api_endpoint, which, get_playbook_path
+from mariner.util import which, get_playbook_path
 
 logger = logging.getLogger(__name__)
 
 
-def make_ansible_command(stderr, stdout, hosts_file, identifier, extra_vars='{}', tags=''):
+def make_ansible_command(hosts_file, identifier, extra_vars='{}', tags=''):
     """
     This utility will compose the command needed to run ansible, capture its
     stdout and stderr to a file
     """
     playbook = get_playbook_path()
     ansible_path = which('ansible-playbook')
-    url = api_endpoint('tasks', identifier, 'completed')
-    command = '{ansible_path} -i {hosts_file} --extra-vars="{extra_vars}" --tags="{tags}" {playbook}'.format(
-            ansible_path=ansible_path,
-            hosts_file=hosts_file,
-            extra_vars=extra_vars,
-            tags=tags,
-            playbook=playbook
-    )
 
-    redirect = '> {stdout} 2> {stderr};'.format(stdout=stdout, stderr=stderr)
-    post = 'curl -X POST {url}'.format(url=url)
-    return "{command} {redirect} {post}".format(command=command, redirect=redirect, post=post)
+    return [
+        ansible_path, '-i', hosts_file,
+        '--extra-vars="%s"' % extra_vars, '--tags="%s"' % tags, playbook
```

Comment:
I think you'll need to json.dumps(extra_vars) here. You might also need to quote the json. So:

```
"--extra-vars='%s'" % json.dumps(extra_vars)
```